Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2790,6 +2790,15 @@ - (YapDatabaseConnection *)editingDatabaseConnection
     return _editingDatabaseConnection;
 }
 
+- (void)yapDatabaseModifiedExternally:(NSNotification *)notification
+{
+    OWSAssert([NSThread isMainThread]);
+
+    DDLogVerbose(@"%@ %s", self.logTag, __PRETTY_FUNCTION__);
+
+    [self resetMappings];
```

Comment:
So, we can't do incremental updates from external modifications, so we just reset our mapping and then reload our view contents.